src/styles/header.scss

```diff
@@ -0,0 +1,58 @@
+@import 'variables';
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+
+  .logo {
+    display: flex;
+  }
+
+  .nav__list {
+    list-style-type: none;
+    display: flex;
+    padding: 0;
+    margin: 0;
+
+    .nav__item {
+      margin-right: 20px;
```

Comment:
The right margin of `.nav__item` should also be a multiple of 4. Adjust `margin-right: 20px;` accordingly.